### DISCUSSION TITLE
A/G contracts

### DISCUSSION THREAD

**Discussion by omidi226** [pacti-org/pacti#345] [General]
Dear Sir/Madam
I would be grateful if you could answer my question.
I found and used operators A/G, but when I want to write two different contracts on component C and apply the conjunction, I can not find out where the conjunction is or if it has not been implemented yet?
My special thanks to your excellent work

**Comment by iincer** [comment]
Hello @omidi226, we are glad to know you have found the project useful! You are right: currently, Pacti implements composition, quotient, and merging. We usually obtain contract conjunction by carrying out case analysis. For example, suppose $\\{C_i\\}$ is a set of contracts for various components under a certain mode of operation, i.e., the assumptions say that the contracts operate at night time. And suppose that $\\{\tilde C_i\\}$ is a set of contracts for the same components that state their responsibilities at night time. If we analyze independently the system using the contracts $C_i$ and then using the $\tilde C_i$, we are implicitly doing what the conjunction operation would do for us.
